dom.bs

```diff
@@ -3420,7 +3420,7 @@ run these steps:
   </ol>
 
  <li><p><a for=set>For each</a> <var>slot</var> of <var>signalSet</var>, <a>fire an event</a>
- named {{HTMLSlotElement/slotchange}}, with its {{Event/bubbles}} attribute set to true, at
+ named <dfn event for=HTMLSlotElement>slotchange</dfn>, with its {{Event/bubbles}} attribute set to true, at
  <var>slot</var>.
```

Comment:
```suggestion
  <li><p><a for=set>For each</a> <var>slot</var> of <var>signalSet</var>, <a>fire an event</a> named
  <dfn event for=HTMLSlotElement><code>slotchange</code></dfn>, with its {{Event/bubbles}} attribute
  set to true, at <var>slot</var>.
```

Reply:
fixed in https://github.com/whatwg/dom/pull/1115/commits/2349d5bebc4b42f7e19454144df413716e6690f7